composer.json

```
@@ -5,8 +5,8 @@
 	"homepage": "https://polylang.pro",
 	"type": "library",
 	"require-dev": {
-		"php": "~7.1",
-		"php-stubs/generator": "^0.8.1"
+		"php": "^7.3 || ^8.0",
+		"php-stubs/generator": "^0.8.4"
```

```suggestion
		"php-stubs/generator": "^0.8"
```
What do you think to allow next patches?